docs/reference/cli/mcpchecker_result.md

```
@@ -0,0 +1,24 @@
+
+Commands for inspecting and analyzing evaluation result files
+
+### Synopsis
+
+Commands for inspecting and analyzing evaluation result files.
+
+These commands operate on the JSON result files produced by 'mcpchecker check'.
+
+### Options
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)